src/g_editor.c

```diff
@@ -4123,6 +4135,39 @@ static void canvas_paste(t_canvas *x)
     }
 }
 
+static void canvas_paste_from_clipboard_text(t_canvas *x)
+{
+    pdgui_vmess("pdtk_get_clipboard_text", "^", x);
+}
```

Comment:
i wonder: is there any specific reason for the roundtrip through the core?
(apart from symmetry with the other menu-entries)

Reply:
No - it's leftover from starting work based on existing menu-entries and functions. thanks